package.json

```diff
@@ -3,14 +3,17 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  "files": [
```

Comment:
It applies to both, so good catch, added `server.js`.

`website.js` and the `website` folder should be moved into a seperate repo 👍